capabilities_test.go

```diff
@@ -8,22 +8,41 @@ import (
 
 var _ = Describe("Capabilities", func() {
 	var capabilities Capabilities
+	proxyType := "manual"
+	proxyAddress := "localhost:8080"
```

Comment:
Using `localhost:8080` implies that the format of this value is important for the implementation. Let's go with `some-http-proxy`.